barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -88,33 +107,52 @@ class ClientIVCTests : public ::testing::Test {
 TEST_F(ClientIVCTests, Full)
 {
     ClientIVC ivc;
-
+    Composer composer;
     // Initialize IVC with function circuit
     Builder function_circuit = create_mock_circuit(ivc);
     ivc.initialize(function_circuit);
+    composer.compute_commitment_key(ivc.prover_fold_output.accumulator->instance_size);
 
+    auto function_vk = composer.compute_verification_key(ivc.prover_fold_output.accumulator);
+    auto verifier_accumulator = std::make_shared<VerifierInstance>();
```

Comment:
can you rename these accumulators to just `foo_verifier_instance` to reflect the fact they aren't being used as accumulators?